src/server.h

```diff
@@ -3336,15 +3340,19 @@ robj *setTypeDup(robj *o);
 #define HASH_SET_COPY 0
 
 
+void dbUpdateKeyWithVolaItemsTracking(serverDb *db, robj *o);
 void hashTypeFreeVolatileSet(robj *o);
-void hashTypeTrackEntry(robj *o, void *entry);
-void hashTypeUntrackEntry(robj *o, void *entry);
-void hashTypeTrackUpdateEntry(robj *o, void *old_entry, void *new_entry, long long old_expiry, long long new_expiry);
+void hashTypeTrackEntry(serverDb *db, robj *o, void *entry);
+void hashTypeUntrackEntry(serverDb *db, robj *o, void *entry);
+void hashTypeTrackUpdateEntry(serverDb *db, robj *o, void *old_entry, void *new_entry, long long old_expiry, long long new_expiry);
+vset *hashTypeGetVolatileSet(robj *o);
```

Comment:
Isn't this an internal detail of the t_hash implementation?  Why is this API exposed?